vars/rollingUpgradePipeline.groovy

```diff
@@ -46,9 +49,11 @@ def call(Map pipelineParams) {
                 steps {
                     script {
                         def tasks = [:]
+                        def email_recipients = groovy.json.JsonOutput.toJson(params.email_recipients)
```

Comment:
Amos, please move this to stage 
``
`stage("Send email for Upgrade from ${base_version}") {
```

Reply:
Ok then, i will send a pr to master with backport label